Common/GPU/D3D11/thin3d_d3d11.cpp

```diff
@@ -983,6 +989,8 @@ Pipeline *D3D11DrawContext::CreateGraphicsPipeline(const PipelineDesc &desc) {
 	std::vector<D3D11ShaderModule *> shaders;
 	D3D11ShaderModule *vshader = nullptr;
 	for (auto iter : desc.shaders) {
+		iter->AddRef();
```

Comment:
Hmm, seems like we'd need a Release() for this in the destructor of the pipeline?

-[Unknown]

Reply:
That is right! Thanks for the catch!